deb/deb.go

```diff
@@ -554,18 +554,27 @@ func createControl(instSize int64, md5sums []byte, info *nfpm.Info) (controlTarG
 		return nil, err
 	}
 
-	filesToCreate := map[string][]byte{
-		"./control":   body.Bytes(),
-		"./md5sums":   md5sums,
-		"./conffiles": conffiles(info),
+	// ensure predefined sort order of these items
+	filesToCreateNames := []string{
```

Comment:
Go `map[string]` does not ensure predefined sort order when iterated via `for ... range ...` and that is the cause of `control.tar.gz` is different size when executed on the same input.